conf/webknossos.latest.routes

```diff
@@ -117,7 +118,7 @@ PATCH         /annotations/:typ/:id/transfer                                   c
 GET           /annotations/:typ/:id/info                                       controllers.AnnotationController.info(typ: String, id: String, timestamp: Long)
 PATCH         /annotations/:typ/:id/makeHybrid                                 controllers.AnnotationController.makeHybrid(typ: String, id: String, fallbackLayerName: Option[String])
 PATCH         /annotations/:typ/:id/downsample                                 controllers.AnnotationController.downsample(typ: String, id: String)
-PATCH         /annotations/:typ/:id/unlinkFallback                             controllers.AnnotationController.unlinkFallback(typ: String, id: String)
+PATCH         /annotations/:typ/:id/unlinkFallback                             controllers.AnnotationController.unlinkFallback(typ: String, id: String, tracingId: String)
```

Comment:
@fm3 Should the `tracingId` be part of the URL as it's done for `editAnnotationLayer`? Just stumbled about this. I still need to adapt the front-end here and I'm wondering where to put the id :thinking: